pom.xml

```diff
@@ -26,13 +26,13 @@
     <properties>
         <revision>1.45</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.164.3</jenkins.version>
         <java.level>8</java.level>
-        <workflow-api-plugin.version>2.30</workflow-api-plugin.version>
         <useBeta>true</useBeta>
```

Comment:
`useBeta` is still required.
```
[ERROR] hudson/plugins/copyartifact/CopyArtifact:671 jenkins/util/VirtualFile.list(Ljava/lang/String;Ljava/lang/String;Z)Ljava/util/Collection; is still in beta
[ERROR] hudson/plugins/copyartifact/CopyArtifact:689 jenkins/util/VirtualFile.readLink()Ljava/lang/String; is still in beta
[ERROR] hudson/plugins/copyartifact/CopyArtifact:696 jenkins/util/VirtualFile.toExternalURL()Ljava/net/URL; is still in beta
[ERROR] hudson/plugins/copyartifact/CopyArtifact:725 jenkins/util/VirtualFile.mode()I is still in beta
```

Reply:
Yes. I do plan to remove the beta mark from at least `list` and `toExternalURL`. I am less sure about `readLink` and `mode`, which are indeed used here though my opinion is that users would be better off not relying on some filesystem-specific features.

Reply:
👍 
I definitely agree that. I wrote that in the documentation: https://github.com/jenkinsci/copyartifact-plugin#recommended-usage-of-copyartifact
But copyartifact should keep backward compatibilities and continues to use `mode` and `readLink`...